docs/analytics/jenkins_jobs.md

```diff
@@ -271,3 +275,40 @@ analytics-configuration/automation/run-automated-task.sh StudentEngagementCsvFil
   --interval-type weekly \
   --interval "$FROM_DATE-$TO_DATE"
 ```
+
+## Problem Response Reports
```

Comment:
Nit: It would be nice to also mention the insights waffle flag for report downloads in this section and/or link to the S3 section of `AWS_setup.md`